core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.component.ts

```diff
@@ -142,6 +142,7 @@ export class QuestionSuggestionReviewModalComponent
   questionId: string | null = null;
   isFirstItem: boolean = true;
   isLastItem: boolean = true;
+  stateContentShouldStayVisibleOnScroll: boolean = true;
```

Comment:
If this value is true by default and never changed, why not inlining it in html?